packages/vite/src/migrations/update-20-5-0/update-resolve-conditions.md

```diff
@@ -0,0 +1,51 @@
+#### Update `resolve.conditions` to include defaults
+
+The `resolve.conditions` option had previously had defaults provided by Vite. These defaults were `['module', 'browser', 'development|production']`.  
```

Comment:
Wording seems off. I'd say something like:

In previous Vite versions, the `resolve.condition` option had a default value of `[...]`. This default was removed in Vite 6, so this migration adds it to your existing configuration to ensure that the behavior remains intact.